rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -610,7 +610,7 @@ SecRule ARGS_NAMES|ARGS|FILES_NAMES "@rx ^\(\s*\)\s+{" \
     phase:2,\
     block,\
     capture,\
-    t:none,t:urlDecode,t:urlDecodeUni,\
+    t:none,t:urlDecodeUni,\
```

Comment:
Probably not an issue, but would this detect some doubly encoded stuff?

Reply:
I don't think that shell is doing URL decode of commands.

Reply:
I don't think we should be urldecoding ARGS.  Per [documentation](https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-(v2.x)#urldecode):
```
... The transformation function should not be used against variables that have already been URL-decoded (such as request parameters) unless it is your intention to perform URL decoding twice!```
So my questions:
- are ARGS_NAMES url decoded already?
- do we need to decode twice ARGS?
- what about FILES_NAMES? Is it already decoded?

Maybe doing a simple test with debuglog enabled could solve this.

Reply:
This is only a first part of the fix (more or less a clean up), please merge it.